bundle/bundle.go

```diff
@@ -23,11 +23,11 @@ type Bundle struct {
 	Keywords         []string               `json:"keywords,omitempty" mapstructure:"keywords"`
 	Maintainers      []Maintainer           `json:"maintainers,omitempty" mapstructure:"maintainers"`
 	InvocationImages []InvocationImage      `json:"invocationImages" mapstructure:"invocationImages"`
-	Images           map[string]Image       `json:"images" mapstructure:"images"`
-	Actions          map[string]Action      `json:"actions,omitempty" mapstructure:"actions"`
-	Parameters       ParametersDefinition   `json:"parameters" mapstructure:"parameters"`
-	Credentials      map[string]Credential  `json:"credentials" mapstructure:"credentials"`
-	Outputs          OutputsDefinition      `json:"outputs" mapstructure:"outputs"`
+	Images           map[string]Image       `json:"images,omitempty" mapstructure:"images,omitempty"`
```

Comment:
mapstructure doesn't support omitempty, so the extra tag should only go on json.

Reply:
ahhh ok!

Reply:
removed